src/pages/Home page/Home.jsx

```diff
@@ -0,0 +1,119 @@
+            <span className="category-card-text">WATERPROOF</span>
+            <span className="category-card-subtext">JACKETS</span>
+          </section>
+          <section className="category-card-badge">
+            <span className="category-card-badge-text">STARTING FROM</span>
+            <span className="badge"><span>₹</span>499</span>
+          </section>
+          <button className="category-card-btn" to="">
+            <NavLink className="btn-link" to="">
+              EXPLORE NOW
+            </NavLink>
+
+          </button>
+        </section>
+
+        {/* <!-- card2 --> */}
+        <section className="category-card">
+          <img alt="please reload" className="category-card-img" src="Images/backpack1.jpg" />
+          <section className="category-card-title">
+            <span className="category-card-text">10-90L</span>
+            <span className="category-card-subtext">BACKPACKS</span>
+          </section>
+          <section className="category-card-badge">
+            <span className="category-card-badge-text">STARTING FROM</span>
+            <span className="badge"><span>₹</span>199</span>
+          </section>
+          <button className="category-card-btn"><NavLink className="btn-link" to="./pages/product/product.html">
+            EXPLORE NOW
+          </NavLink> </button>
+        </section>
+
+        {/* <!-- card 3 --> */}
+
+        <section className="category-card">
+          <img alt="please reload" className="category-card-img" src="Images/gears.jpg" />
+          <section className="category-card-title">
+            <span className="category-card-text">TREKKING</span>
+            <span className="category-card-subtext">Gears</span>
+          </section>
+          <section className="category-card-badge">
+            <span className="category-card-badge-text">STARTING FROM</span>
+            <span className="badge"><span>₹</span>199</span>
+          </section>
+          <button className="category-card-btn"><NavLink className="btn-link" to="./pages/product/product.html">
+            EXPLORE NOW
+          </NavLink> </button>
+        </section>
+
+        {/* <!-- card4 --> */}
+
+        <section className="category-card">
+          <img alt="please reload" className="category-card-img" src="Images/jackets.jpg" />
+          <section className="category-card-title">
+            <span className="category-card-text">CAMPING</span>
+            <span className="category-card-subtext">TENTS</span>
+          </section>
+          <section className="category-card-badge">
+            <span className="category-card-badge-text">STARTING FROM</span>
+            <span className="badge"><span>₹</span>499</span>
+          </section>
+          <button className="category-card-btn"><NavLink className="btn-link" to="./pages/product/product.html">
+            EXPLORE NOW
+          </NavLink> </button>
+        </section>
+
+      </section>
+    </div>
```

Comment:
same with these cards, just make on JSX template and map over it

Reply:
Done